Django_HW/create/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from users.models import User
+from books.models import Book
+from purchases.models import Purchase
+
+
+class UserForm(forms.ModelForm):
```

Comment:
nit: better to use builtin form from auth, there are a lot of useful functionality is already implemented there
```
from django.contrib.auth.forms import UserCreationForm

class CustomUserCreationForm(UserCreationForm):
    class Meta:
        model = get_user_model()
        fields = UserCreationForm.Meta.fields + ('age', )  # if you want to add more fields

```